app/controllers/hideout_controller.rb

```diff
@@ -0,0 +1,94 @@
+class HideoutController < ApplicationController
+  def create
+    # Needs owner_email and name as params.
+    # name must be lesser than 20 characters and greater than 3 characters
+    # owner must not already be in a hideout
+    # return the hideout as an object
+    owner_email = params[:owner_email]
+    name = params[:name]
```

Comment:
Does this read the body of the post request when it is in `json`?

Reply:
yes